.eslintrc.yaml

```diff
@@ -1,16 +1,29 @@
-extends: 'airbnb'
+env:
+  node: true
+
+extends:
+  - eslint:recommended
+  - airbnb
+
+globals:
+  uneval: false
+
 plugins:
   - mozilla
 
+root: true
+
 rules:
-  indent: [1, 2, {SwitchCase: 1}]
-  mozilla/balanced-listeners: [0]
-  mozilla/no-aArgs: [1]
-  mozilla/this-top-level-scope: [1]
-  no-console: [1]
-  import/no-unresolved: [0]
+  import/newline-after-import: off
+  import/no-extraneous-dependencies: off
+  import/no-unresolved: off
 
-globals:
-  require: false
-  exports: true
-  uneval: false
```

Comment:
Add-on SDK add-ons aren't equivalent to node though, so this is actually incorrect. We should keep this as it was.